handler/artifact_upload_handler.py

```diff
@@ -45,24 +48,27 @@ def _check_token(self, token):
         # Reduce the risk of timing analysis attacks.
         return hmac.compare_digest(self.hash_expected, hash_result)
 
-    def _index_channel(self):
+    def _index_channel(self, channel):
         """
-        Index the conda channel at `self.write_path` to ensure the channel
-        remains consistent.
+        Index the conda channel to ensure that it remains consistent.
```

Comment:
In general, I find it unclear what `channel` refers to. For example conda-forge is a channel, but also conda-forge/linux64 is a channel (which is how you've used the term here). 
There is a risk of confusion. Is there better terminology to differentiate them?

Reply:
Yup

Reply:
We refer to both as channel. In the conda codebase they refer to linux64 and noarch as `subdir` but I've also seen it be `platform dir`